parsec/core/gui/central_widget.py

```diff
@@ -32,7 +34,7 @@ class CentralWidget(QWidget, Ui_CentralWidget):
         "mountpoint.remote_error",
         "mountpoint.unhandled_error",
         "sharing.updated",
-        "fs.entry.file_update_conflicted",
+        "fs.entry.file_conflict_resolved",
```

Comment:
I can't see where this event is triggered (didn't find any reference of `fs.entry.file_update_conflicted` either...)
Are you sure this event is still in use ? or is it for future use ?

Reply:
file_update_conflicted does not exist anymore, but file_conflict_resolved is a new event (https://github.com/Scille/parsec-cloud/pull/1095)